Updated with info about the order (#2751)
umbraco/UmbracoDocs: Umbraco-Cloud/Set-Up/Manage-Environments/index.md

On that page you will be able to add or remove an environment, if you wish to remove one you will be prompted to type in the environment alias, so either 'Development' or 'Staging'.

> Be aware that the order that the enviroments are being added in are as followed. You will need to have a development environment, before you can have a staging environment.

Reviewer: ```suggestion
Be aware that there is a specific order that the environments are being added in. You will need to have a Development environment before you can have a Staging environment.
```

> If you have both development and staging environment and needs to remove the development environment, then you will need to remove the staging environment first before you can remove the developmentenvironment. 

Reviewer: ```suggestion
If you have both a Development and a Staging environment and need to remove the Development environment, then you will need to remove the Staging environment first before you can remove the Development environment. 
```
